src/components/LayoutTree.tsx

```diff
@@ -364,16 +370,24 @@ const LayoutTree = () => {
         state.cursor.length - (hasChildren(state, head(state.cursor)) ? 2 : 3)
       : 0,
   )
+  const scrollTop = scrollTopStore.useState()
 
   const [bulletWidth, setBulletWidth] = useState<number | undefined>(0)
+  const toolbarHeightRef = useRef(0)
+  const distanceFromTop = useRef<number | undefined>(0)
 
   // set the bullet width only during drag or when simulateDrop is true
   useLayoutEffect(() => {
     if (dragInProgress || testFlags.simulateDrop) {
       const bullet = ref.current?.querySelector('[aria-label=bullet]')
       setBulletWidth(bullet?.getBoundingClientRect().width)
+
+      const toolbar = document.querySelector('#toolbar')
+
+      if (toolbar) toolbarHeightRef.current = toolbar?.getBoundingClientRect().height
```

Comment:
```suggestion
      if (toolbar) toolbarHeightRef.current = toolbar.getBoundingClientRect().height
```

Minor adjustment. We don't need the conditional `?` since the `if` condition has already assured us that the `toolbar` is defined.

Side note, @raineorshine: This ESLint rule could check (and auto-fix) for these: https://typescript-eslint.io/rules/no-unnecessary-condition/